src/utils/quick-open.js

```diff
@@ -54,7 +54,7 @@ export function parseLineColumn(query: string) {
 export type QuickOpenResult = {|
   id: string,
   value: string,
-  title: string,
+  title: string | any,
```

Comment:
Do we expect the title to be anything else than a string?

Reply:
Yea actually I was puting <span></span> into it, and if I convert span to a string it renders as Object Object.